examples/hata_basic_queue/bot.py

```diff
@@ -172,19 +181,32 @@ async def skip(ctx):
 @bot.commands
 @checks.guild_only()
 async def pause(ctx):
-    """Pause the current song."""
+    """Pause the current track."""
     await ctx.client.data.lavalink.pause(ctx.guild.id)
     await ctx.reply("Paused player")
 
 
 @bot.commands
 @checks.guild_only()
 async def resume(ctx):
-    """Resume playing the current song."""
+    """Resume playing the current track."""
     await ctx.client.data.lavalink.resume(ctx.guild.id)
     await ctx.reply("Resumed player")
 
 
+@lightbulb.check(lightbulb.guild_only)
+@lightbulb.command(name='now-playing', aliases=['np'])
```

Comment:
Hata totally uses lightbulb :D
Fix this to Hata's command framework.

Reply:
Fixed :P

Reply:
This has not been fixed.